doc/getting-started.md

```diff
@@ -0,0 +1,344 @@
+        "reqSigs": 1,
+        "type": "pubkey",
+        "addresses": [
+          "D5TjotnkoXekNQBCn54CBWxjEzCJMMe7WS"
+        ]
+      }
+    }
+  ]
+}
+```
+
+`vin` and `vout` will give you information about transaction inputs and outputs.
+
+## Node configuration
+
+A server contain a lot of parameter to configure, from memory limits, to RPC credentials, ports... There is two ways to prepare your perfect node !
+
+Using directly `dogecoind` options, `--help` will display all configuration parameters.
+
+**Command example :**
+```bash
+$ dogecoind -testnet -datadir=/path/to/dir -rpcuser=Shibetoshi -rpcpassword=Nakamoto
+```
+
+Otherwise, a node is configurable through a `dogecoin.conf` file. Create it in the directory defined with the `datadir` setting, `$HOME/.dogecoin` by default, or specify the file location with `-conf`.
+
+**dogecoin.conf example :**
+```
+testnet=1
+datadir=/path/to/directory
+rpcuser=Shibetoshi
+rpcpassword=Nakamoto
+```
+You can see a more concrete example [here](/contrib/debian/examples/dogecoin.conf).
+
+### Mainnet, testnet and regnet
+
+To perform tests, for example to create a blockchain based application, you may need a sandbox environment. Multiple networks exists for this purpose.
+
+**Mainnet** : The main network where real transaction operate.  
+**Testnet** : The test network, with peers.  
+**Regnet** : The regression network, to test without peers and create blocks on-demand.
+
+*Mainnet* is the network used by default. To enable *testnet*, with `dogecoind` use the `-testnet` option and `testnet=1` as `dogecoin.conf` field.
+
+To enable *regnet*, use `-regtest` option.
+
+> **Tips :** Think to specify the network when you use `dogecoin-cli`.
+
+### RPC credentials
+
+A username and a password is needed to control the RPC interface. When no credentials are provided, dogecoin use a [special cookie](https://bitcoin.org/en/release/v0.12.0#rpc-random-cookie-rpc-authentication) to randomly generate it.
+
+For security reasons, because the RPC server may process sensitive data like private keys, you can define your own identifiers using `rpcuser` and `rpcpassword` settings.
+
+### Ports
+
+A node exposes up to 2 ports. One mandatory port for the **Peer to Peer Network (P2P)** to communicate with other nodes, a second optional port expose the RPC API.
+
+| Function | mainnet | testnet | regtest |
+| :------- | ------: | ------: | ------: |
+| P2P      |   22556 |   44556 |   18444 |
+| RPC      |   22555 |   44555 |   18332 |
+
+To configure them use `port` and `rpcport` settings.
+
+### Memory
+
+Dogecoin Core use important ressources for some environment, it may be interesting to restrict them.
+
+Multiple parameter to play with, listed here with their default values :
+```bash
+$ dogecoind -dbcache=450 -maxmempool=300 -maxconnections=125
+```
+- **dbcache :** UTXO database cache size.
+- **maxmempool :** Limit memory pool size.
+- **maxconnections :** Maximum peer connections.
+
```

Comment:
I think that none of this helps with getting started.

Reply:
It can be useful to deal with memory configuration when you're starting your node no ?

Reply:
I'm not sure. changing `maxmempool` can have effects to what fees your node is willing to relay. Is that something for beginners? `dbcache` - I'd only tune that in combination with debug=bench and some trial and error, which is not something I would explain in a getting started guide.

Reply:
It was for low memory environment (I looked on it because I had some issues with my raspberry).

Reply:
I think that should be in a separate guide then? 1.21 has [reduce-memory.md](https://github.com/dogecoin/dogecoin/blob/1.21-dev/doc/reduce-memory.md)

Reply:
I missed it. Do you think I should delete the section entirely, or keep it with a sentence to link it to this file ?

Reply:
Let's backport it into 1.14.5 (test it to be correct, remove whatever we don't have yet) and then we can link to it here as a "for information on how to reduce the memory footprint, see *link*"?

I'll work on the backport right now.

Reply:
#2664 adds that document to the 1.14 tree.

Reply:
Linked.